### DISCUSSION TITLE
Dexcom G7 vs One+ in xDrip

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#3677] [Q&A]
I'm very prone to return to Dexcom G6, due to the issue I've encountered using the G7, which are not related to xDrip+ but to the sensor itself: essentially, weak bluetooth signal and jumpy values.
My diabetologist suggested to try the Dexcom One+ instead; I understand that it's "more basic" if compared to G7 in terms of alarm predicivity but I think it's something related to the app, so if using xDrip+ the experience should be the same from this point of view.
Is Dexcom One+ fully functional in xDrip+?

**Comment by Navid200** [comment]
Yes, it is.
https://navid200.github.io/xDrip/docs/Dexcom/G7.html

**Reply by dagord** [reply]
Hello,
thank you for your quick answer.
I already read that page, but since the "hardware" is pretty much the same, I'm wondering if I should expect the same "experience" I had with the G7.
E.g., I read that One+ doesn't support calibrations, but it's a limitation of the sensor itself or a limitation of the Dexcom app that I can get around using xDrip? Since the intermittent BT connectivity is main reason that is pushing me to return to G6, is the One+ worth a try? I understand that this question is not directly related to xDrip but any clarification would help me. :)
Thanks!

**Reply by Navid200** [reply]
I see,

Unfortunately, I have never had a One+.  So, I cannot tell you if it accepts calibration or not.

If intermittent connectivity is the reason you are not happy with G7, I would not expect improvements going to One+.
I don't know why you expect improvements in that front going to G6.

Intermittent connectivity that cannot be fixed is usually caused by the phone manufacturer.

**Reply by dagord** [reply]
I used the G6 for a couple of years and it was rock solid from this point of view.
The experience with G7 with the official Dexcom 7 app is very poor (and I'm using a Redmi Note 10, so one of the very few certified smartphones!); with xDrip+ the connection is much more stable but not as reliable as with the G6 :(

**Reply by Navid200** [reply]
Alright
I did not expect that.

I don't believe One+ has any enhancements that G7 does not.  But, that is not based on any measurements or publications.  
Hopefully someone with experience with both will chime in to provide a good perspective.

**Reply by Navid200** [reply]
When you use G7, and have intermittent connectivity, do you sometimes  see a red note on the status page saying slowest wake or something like that?

**Reply by dagord** [reply]
Well, I already opened a thread about this some time ago and you helped me a lot.
https://github.com/NightscoutFoundation/xDrip/discussions/3454

It seems it's an issue with Xiaomi phones but, as I wrote, in my case the G6 was rock solid

**Reply by Navid200** [reply]
Oh, Redmi is Xiaomi.  Sorry, I missed that
